src/resources/i18n/translations/en.json

```
@@ -359,15 +358,15 @@
 	"Latest episode": "Latest episode",
 	"Latest": "Latest",
 	"Leaving App": "Leaving…",
-	"License": "Copylefted libre license",
+	"License": "License",
```

```suggestion
	"License": "Copylefted libre license",
```
Make the most of communicating this difference wherever possible.

---

👍 

This translation goes on our Terms of Service screen. Basically, for mostly marketing purposes we prefer "Free and open source" for English. For more detailed descriptions of the app and its license, like on the ToS, we can go with "Copylefted libre license"

---

People understanding the terms of service is important. Using a different term there is a red flag.
The terms of service should be unambiguous, and "free and open source" is not that.